pkg/plugins/golang/v4/scaffolds/internal/templates/webhooks/webhook_suitetest.go

```diff
@@ -208,7 +245,7 @@ var _ = BeforeSuite(func() {
 	Expect(cfg).NotTo(BeNil())
 
 	scheme := apimachineryruntime.NewScheme()
-	err = AddToScheme(scheme)
+	err = %s.AddToScheme(scheme)
```

Comment:
This produces an error when the "legacy" path is used, as there is no import alias required then.

Reply:
Good catcher, 
we should keep as before if is legacy.